arroyo/processing/processor.py

```diff
@@ -223,6 +224,12 @@ def _run_once(self) -> None:
             try:
                 start_poll = time.time()
                 self.__message = self.__consumer.poll(timeout=1.0)
+
+                if self.__message is not None:
+                    self.__metrics_buffer.increment(
+                        ConsumerTiming.CONSUMER_MESSAGE_LATENCY,
+                        time.time() - self.__message.timestamp.timestamp(),
```

Comment:
I think it might make sense to pull the `time.time()` out and put it before both of these metric calls. Otherwise you are including this logic as part of the consumer poll time.